tools/operator/testclient/operations/invokequery.go

```diff
@@ -228,8 +261,8 @@ func (i InvokeQueryUIObject) invokeConfig(channelName string, args []string) err
 func (i InvokeQueryUIObject) invokeQueryTransactions(invokeQueryObjects []InvokeQueryUIObject) error {
 
 	var wg sync.WaitGroup
-	var statusError error
 	pteMainPath := paths.PTEPath()
+	c := make(chan Result, 3*len(invokeQueryObjects))
```

Comment:
3 * len( ... )
can you add comment to explain what is going on here? why 3? is that the maximum size of <what> that could be possibly needed? Or should we instead base this on the size of something?

Reply:
we removed this in favor of a different error handling approach.